### ISSUE TITLE
[ICON REQUEST]: Google Colab

### ISSUE THREAD

**Issue by XI4684-AdithyaShankaran** [devicons/devicon#2228]
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Popular service for using python kernal notebook. its similar to jupyter

### Links and sources

https://colab.research.google.com/

#### iconname-version-name
<img width="100px" src="https://colab.research.google.com/img/colab_favicon_256px.png" ></img>
Google Colabotary

**Comment by canaleal** [comment]
Hey @XI-AdithyaShankaran, I'll be closing this issue as there's already PR for this icon. 

https://github.com/devicons/devicon/pull/2055

**Comment by Snailedlt** [comment]
@canaleal This issue probably shouldn't be closed, since the PR is not yet merged and out in production

**Comment by canaleal** [comment]
Alright sounds good, my mistake. Will add the develop tag once the PR is merged.

**Comment by XI-AdithyaShankaran** [comment]
> Alright sounds good, my mistake. Will add the develop tag once the PR is merged. 

When will it be seen in prod n website if i search colab i couldn't see it yet

**Comment by canaleal** [comment]
@XI-AdithyaShankaran

The Icon will appear in production after it has been merged into the develop branch and included in the new release. Currently, we are on release 2.16.0.

**Comment by XI-AdithyaShankaran** [comment]
> @XI-AdithyaShankaran
> 
> The Icon will appear in production after it has been merged into the develop branch and included in the new release. Currently, we are on release 2.16.0.
> 
> Currently, the website only displays icons in production. However, you can use icons from the develop branch.

Thank you for your kind response. Further i have recented issued some other important icon perspective to development aspect. Everyone knows future is LLMs and Generative AI. So i have asked icons for OpenAI, ChatGPT (product of OpenAI), Gemini, Bard, Mocrosoft Copilot and Bing, Azure OpenAI (service of open ai used for organisation for api purpose).
